Lib/defconQt/fontView.py

```diff
@@ -782,11 +784,12 @@ def __init__(self, font):
 
         menuBar = self.menuBar()
         # TODO: make shortcuts platform-independent
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&New…", self.newFile, QKeySequence.New)
-        fileMenu.addAction("&Open…", self.openFile, QKeySequence.Open)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&New…"), self.newFile, QKeySequence.New)
+        fileMenu.addAction(self.tr("&Open…"), self.openFile,
+                           QKeySequence.Open)
         # recent files
-        self.recentFilesMenu = QMenu("Open &Recent…", self)
+        self.recentFilesMenu = QMenu(self.tr("Open &Recent"), self)
```

Comment:
I forgot to mention this in the commit message, I removed it on purpose because it does not require additional user input. Gimp doesn't have an ellipsis there either.